docs/install-w-machine.md

```diff
@@ -1,231 +1,173 @@
 
-2. Create a VirtualBox machine called `local` on your system.  
+2. Optional: To conserve system resources, stop any virtual machines you are not using. For example, to stop the VM named `default`, enter:
 
-		$ docker-machine create -d virtualbox local
-		INFO[0000] Creating SSH key...                          
-		INFO[0000] Creating VirtualBox VM...                    
-		INFO[0005] Starting VirtualBox VM...                    
-		INFO[0005] Waiting for VM to start...                   
-		INFO[0050] "local" has been created and is now the active machine.
-		INFO[0050] To point your Docker client at it, run this in your shell: eval "$(docker-machine env local)"
+		$ docker-machine stop default
 
-3. Load the `local` machine configuration into your shell.
+3. Create and run a VM named `manager`.  
 
-		$ eval "$(docker-machine env local)"
+		$ docker-machine create -d virtualbox manager
 
-4. Generate a discovery token using the Docker Swarm image.
+3. Create and run a VM named `agent1`.  
 
-	The command below runs the `swarm create` command in a container. If you
-	haven't got the `swarm:latest` image on your local machine, Docker pulls it
-	for you.
+		$ docker-machine create -d virtualbox agent1
 
-		$ docker run swarm create
-		Unable to find image 'swarm:latest' locally
-		latest: Pulling from swarm
-		de939d6ed512: Pull complete
-		79195899a8a4: Pull complete
-		79ad4f2cc8e0: Pull complete
-		0db1696be81b: Pull complete
-		ae3b6728155e: Pull complete
-		57ec2f5f3e06: Pull complete
-		73504b2882a3: Already exists
-		swarm:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
-		Digest: sha256:aaaf6c18b8be01a75099cc554b4fb372b8ec677ae81764dcdf85470279a61d6f
-		Status: Downloaded newer image for swarm:latest
-		fe0cc96a72cf04dba8c1c4aa79536ec3
+3. Create and run a VM named `agent2`.  
 
-	The `swarm create` command returned the  `fe0cc96a72cf04dba8c1c4aa79536ec3`
-	token.
+		$ docker-machine create -d virtualbox agent2
 
-5. Save the token in a safe place.
+Each command checks for a local copy of the latest VM image called boot2docker.iso. If the latest copy isn't available, Docker Machine downloads the latest image from Docker Hub. Then, Docker Machine uses boot2docker.iso to create a VM that automatically runs Docker Engine.
 
-	You'll use this token in the next step to create a Docker Swarm.
+## Create a Swarm discovery token
 
+1. Connect the Docker Client on your computer to the Docker Engine running on `manager`.
 
-## Launch the Swarm manager
+        $ eval $(docker-machine env manager)
 
-A single system in your network is known as your Docker Swarm manager. The swarm
-manager orchestrates and schedules containers on the entire cluster. The swarm
-manager rules a set of agents (also called nodes or Docker nodes).
+    The client will send the `docker` commands in the following steps to the Docker Engine on on `manager`.
```

Comment:
This sentence is awkward.  "in the following steps" is unnecessary and two "on on"

```
Once you configure the client, it sends `docker` CLI commands to the Docker Engine on the Swarm `manager` node.
```